src/utils/errors.ts

```diff
@@ -49,15 +49,16 @@ export const fallbackErrorMessage = (error, type, detail) => {
     case 'add-activity':
       text += ' when submitting your post';
       break;
-    case ('add-reaction', 'add-child-reaction'):
+    case 'add-reaction':
+    case 'add-child-reaction':
       text += ' when submitting your ' + detail.kind;
       break;
-    case ('delete-reaction', 'delete-child-reaction'):
+    case 'delete-reaction':
+    case 'delete-child-reaction':
       text += ' when removing your ' + detail.kind;
       break;
     default:
   }
 
-  text += '. Is your internet working?' + suffix;
-  return text;
+  return text + '. Is your internet working?' + suffix;
```

Comment:
Template literals might look cleaner but this is fine with me as long as it's fine with you. :D

Reply:
yep fixed